CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased
 
 * Removed the validation of a subtype change in integrations and scripts from **validate**.
+* Added a validation to the **validate** command to make sure we are using the https protocol in the defaultvalue of the url parameter.
+* Added a check to the **lint** command for finding hard coded usage of the http protocol.
```

Comment:
just finding, or fixing too?

Reply:
Some times we cannot use the https protocol, so fixing it won't solve it